recipes-devtools/cst/cst_git.bb

```diff
@@ -16,9 +16,9 @@ inherit kernel-arch
 #SECURE_PRI_KEY = "/path/srk.pri"
 #SECURE_PUB_KEY = "/path/srk.pub"
 
-SRC_URI = "git://source.codeaurora.org/external/qoriq/qoriq-components/cst;nobranch=1 \
+SRC_URI = "git://source.codeaurora.org/external/qoriq/qoriq-components/cst;protocol=https;nobranch=1 \
```

Comment:
Please drop the protocol change.

Reply:
prefer https protocol as we got trouble with proxy/firewalls before.
I assume it is acceptable as some other pkgs use https protocol.
$ cd meta-freescale
$ git grep "source.codeaurora" |wc -l
70
$ git grep "source.codeaurora.*protocol=http" |wc -l
29

Reply:
This is what the mirror backend of OpenEmbedded is intended to solve, so please don't add it and instead add a mirror item.

Reply:
drop protocol change now.